### ISSUE TITLE
NVDA does not read characters in edit field in VS Code quick pick

### ISSUE THREAD

**Issue by isidorn** [nvaccess/nvda#13037]
Latest VS Code insiders `1.63.0`
NVDA version `2021.2`

1. Open VS Code insiders, start NVDA
2. Press `Ctrl + P` -> you get the quick pick dialog
3. Type "hello"
4. Use left / right arrows to navigate across the hello word. NVDA is not reading characters 🐛 

If you go to the editor you will see that NVDA nicely reads characters when using arrow navigation.
This works fine with other screen readers like Orca or VoiceOver.

I apologise for filling a very VS Code specific bug. If you are interested in looking more into this let me know and I can help from the VS Code side.
Thank you very much 🙏 

Originally found by @neurrone
fyi @feerrenrut

**Comment by leonardder** [comment]
I will look into this shortly.

**Comment by leonardder** [comment]
I can also reproduce this with Narrator. Furthermore, it really seems that the list items gain focus. I'm not sure whether this has to be fixed on NVDA"s end, as the focus stays on the list item when the caret position changes in the edit box, at least, it seems so.

**Comment by isidorn** [comment]
I have commented on the VS Code issue, let's continue the discussion there. Thanks!

**Comment by leonardder** [comment]
Closing as per above comments. Interested people can follow the progress in the VS Code issue.
